apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -508,6 +508,8 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         if check_indentation:
             help_text += "<br><b>Your answer should be indented. </b> Indent your tiles by dragging them horizontally in the answer area."
 
+        help_text += "<br>Keyboard Controls: Arrows to navigate. Enter to select, escape to deselect blocks."
```

Comment:
Nit, consistency:
```suggestion
        help_text += "<br>Keyboard Controls: Arrows to navigate; Enter to select; Escape to deselect blocks."
```

Reply:
Also, it may be better (since we're talking about accessibility) to [replace the `<br>` tags with paragraphs (`<p>...</p>`)](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML), here and in the other items that precede it.